.storybook/Absolute.js

```diff
@@ -0,0 +1,113 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import styled from 'styled-components'
+import { Absolute, Box, Card, Flag, Flex, Icon, Image, Text } from '../src'
+
+const RelativeBox = styled(Box)`
+  position: relative;
```

Comment:
Would be good to add a `Relative` component in a follow up PR as discussed in #318

Reply:
That's my next PR : )

Reply:
I would vote that `Relative` should be part of this PR.

Reply:
We discussed in the meeting and think it's fine for Relative to be a follow up PR